cmd/download.go

```diff
@@ -162,6 +162,19 @@ func newDownload(flags *pflag.FlagSet, usrCfg *viper.Viper) (*download, error) {
 	d.apibaseurl = usrCfg.GetString("apibaseurl")
 	d.workspace = usrCfg.GetString("workspace")
 
+	if d.uuid == "" {
+		if d.slug == "" {
+			if _, slug, ok := trackAndSlugFromCwd(d.workspace); ok && slug != "" {
```

Comment:
oh, the `!= ""` check is, strictly speaking unnecessary. If we have gotten to this point, then `d.slug` is already `""` and setting it from `""` to `""` does nothing.